### DISCUSSION TITLE
What script actually installs the actions runner?

### DISCUSSION THREAD

**Discussion by rgl** [actions/virtual-environments#4337] [Installed software]
I'm trying to find the script that actually install the actions runner inside this repository, but the obvious search for [runner/releases](https://github.com/actions/virtual-environments/search?q=runner%2Freleases) does not return anything, nor manually browsing https://github.com/actions/virtual-environments/tree/main/images/linux/scripts/installers, so I'm quite puzzled.

Can you point me to the script that is actually installing the actions runner from https://github.com/actions/runner/releases?

**Comment by miketimofeev** [comment]
@rgl I'm afraid we can't provide any details here as there is an internal tool that provisions hosted VMs and installs the agent on them as a part of the provisioning process.

**Comment by rgl** [comment]
Can that disclaimer be added to this repository README?
